### PR TITLE
Add arbitrage trading mode

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/243
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/244

### CODE REVIEW

File: Trading/Mode/arbitrage_trading_mode/arbitrage_trading_mode.py [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+            }
+            await self.submit_trading_evaluation(cryptocurrency=self.trading_mode.cryptocurrency,
+                                                 symbol=self.trading_mode.symbol,
+                                                 time_frame=None,
+                                                 state=arbitrage_container.state,
+                                                 data=data)
+
+    async def _trigger_arbitrage_secondary_order(self, arbitrage, filled_order, filled_quantity_before_fees):
+        arbitrage.passed_initial_order = True
+        now_buying = arbitrage.state is EvaluatorStates.SHORT
```
Comment (Herklos) [inline Trading/Mode/arbitrage_trading_mode/arbitrage_trading_mode.py:357]: `is not EvaluatorStates.SHORT` ?

Comment (GuillaumeDSM) [reply Trading/Mode/arbitrage_trading_mode/arbitrage_trading_mode.py:357]: in this context an ```EvaluatorStates.SHORT``` arbitrage is when you first sell to buy later, in this case, the secondary order is a buy

**Comment by Herklos** [comment]
Really great job :100:

File: Trading/Mode/arbitrage_trading_mode/arbitrage_trading_mode.py (hunk 1 above)
Comment (Herklos) [reply Trading/Mode/arbitrage_trading_mode/arbitrage_trading_mode.py:357]: Oh ok, can you explain it with a comment above please ?

**Pull request merged**